Spring-Boot/Spring-Boot-Academy/src/test/java/tw/waterball/judgegirl/springboot/academy/controllers/ExamControllerTest.java

```diff
@@ -557,6 +556,17 @@ void testAnswerQuestionConcurrentlyWithSubmissionQuotas() throws Exception {
         });
     }
 
+    @Test
+    void GivenExamSavedAndGetExam_WhenDeleteExamById_ShouldExamBeNotEmpty() throws Exception {
```

Comment:
bad name, the testcase's name should be **narrative**

please improve it yourself for a while